doc/concepts/blockchain.md

```diff
@@ -81,24 +81,22 @@ duration of an epoch.
 
 ## Leader
 
-Leader are an abstration related to the specific actor that have the ability
+Leader are an abstraction related to the specific actor that have the ability
 to create block; In OBFT mode, the leader just the owner of a cryptographic
-key, whereas in GenesisPraos mode, the leader is a stake pool.
+key, whereas in Genesis-Praos mode, the leader is a stake pool.
 
 ## Transaction
 
-Transaction forms the cornerstone of the blockchain, and is one type of fragment
-and also the most frequent one.
+Transaction forms the cornerstone of the blockchain, and is one type of fragment and also the most frequent one.
```

Comment:
It's actually pretty common to stick to 80 columns per line. I think this change is not necessary. and is adding noise on the history.

Reply:
ok understood, sorry, will re-PR